modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -1,59 +1,60 @@
-= Policy Level
+= Resource Level Policies
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: policy, custom, ootb, offline, resource level
 
-Policies are by default applied to the entire API, unless configured otherwise. However, you can implement an additional level of granularity for policies, in which access is controlled based on the criteria. These are called _resource-level policies_. 
+Policies are by default applied to the entire API, unless configured otherwise. However, you can implement an additional level of granularity for policies, in which access is controlled based on a criteria. These are called _resource-level policies_. 
```

Comment:
```suggestion
Unless they are configured otherwise, policies are by default applied to the entire API. However, you can implement an additional level of policy granularity, one in which access is controlled based on a criteria. Policies with this granularity are called _resource-level policies_. 
```

Reply:
Done! Thanks!